src/Behavioral.Automation/Services/PrintValuesHelper.cs

```diff
@@ -8,7 +8,8 @@ public static class PrintValuesHelper
     {
         public static string GetPrintableValues(this IEnumerable<ITableRowWrapper> rows)
         {
-            return rows.SelectMany(x => x.CellsText).Aggregate((x, y) => $"{x}, {y}");
+            var values = rows.Aggregate("\r\n|", (current, row) => current + (row.CellsText.Aggregate((x, y) => $" {x} | {y}") + " |\r\n |"));
```

Comment:
`Aggregate `method doesn't apply for empty or null collections and will throw exception in this case. I guess we should check `rows` for null/empty and throwing a readable exception or empty string in this case

Reply:
Added check which returns "Row collection is empty" string